### DISCUSSION TITLE
How can I get Juggluco to display mmol instead of mgdl?

### DISCUSSION THREAD

**Discussion by kristinsmotz** [maheini/FreeStyle-Libre-3-patch#43] [Q&A]
We just installed Juggluco and successfully took over a FSL3 sensor from the official app. But juggluco is displaying numbers in Mgdl values and not mmol which is what we use. We want to connect to AAPS for looping, but don't know if the values being in mgdl will cause problems there. Will numbers be automatically translated or can I adjust this in juggluco to send mmol numbers instead?

**Comment by maheini** [comment]
Hi @kristinsmotz 
You can switch units within Juggluco - just open the Settings and scroll all the way UP. Here you'll be able to choose your BG unit. Hope it will help you further. If you got any other question, feel free to ask 😄

**Reply by kristinsmotz** [reply]
Thank you! I don't know why I hadn't tried scrolling 🤣🤣
